sdk/media/azure-media-lva-edge/README.md

```diff
@@ -0,0 +1,127 @@
+
+Use the client library for Live Video Analytics on IoT Edge to:
+
+- Simplify interactions with the [Microsoft Azure IoT SDKs](https://github.com/azure/azure-iot-sdks) 
+- Programatically construct media graph topologies and instances
+
+[Package (PyPi)][package] | [Product documentation][doc_product] | [Direct methods][doc_direct_methods] | [Media graphs][doc_media_graph] | [Source code][source] | [Samples][samples]
+
+## Getting started
+
+### Install the package
+
+Install the Live Video Analytics client library for Python with pip:
+
+```bash
+pip install azure-lva-edge
+```
+### Prerequisites
+
+* Python 2.7, or 3.5 or later is required to use this package.
+* You need an [Azure subscription][azure_sub], and a [IOT device connection string][iot_device_connection_string] to use this package.
+
+### Creating a graph topology and making requests
+Please visit the [Examples](#examples) for starter code
+## Key concepts
+
+### Graph Topology vs Graph Instance
+A graph topology is essentially the blueprint or template of a graph. It defines the parameters of the graph using placeholders as values for them. A graph instance references a graph topology and specifies the parameters. This way you are able to have multiple graph instances referencing the same topology but with different values for parameters. For more information please visit [Media graph topologies and instances][doc_media_graph] 
+
+### CloudToDeviceMethod
+
+The `CloudToDeviceMethod` is part of the azure-iot-hub sdk. This method allows you to communicate one way notifications to a device in your iot hub. In our case we want to communicate various graph methods such as `MediaGraphTopologySetRequest` and `MediaGraphTopologyGetRequest`. To use `CloudToDeviceMethod` you need to pass in two parameters: `method_name` and `payload`. `method_name` should be the name of the media graph request you are sending. Each media graph request has a property called `method_name`. For example, `MediaGraphTopologySetRequest.method_name`. For the second parameter `payload` send the entire serialization of the media graph request. For example, `MediaGraphTopologySetRequest.serialize()`
+
+## Examples
+
+### Creating a graph topology
+To create a graph topology you need to define parameters, sources, and sinks.
+```
+#Parameters
+user_name_param = MediaGraphParameterDeclaration(name="rtspUserName",type="String",default="dummyusername")
+password_param = MediaGraphParameterDeclaration(name="rtspPassword",type="String",default="dummypassword")
+url_param = MediaGraphParameterDeclaration(name="rtspUrl",type="String",default="rtsp://www.sample.com")
```

Comment:
might want to reconsider the default value. maybe use the existing rtspsim link as default.

rtsp://rtspsim:554/media/camera-300s.mkv